ci/test/00_setup_env_native_tsan.sh

```diff
@@ -10,6 +10,7 @@ export CONTAINER_NAME=ci_native_tsan
 export DOCKER_NAME_TAG=ubuntu:16.04
 export PACKAGES="clang llvm python3-zmq qtbase5-dev qttools5-dev-tools libevent-dev bsdmainutils libboost-system-dev libboost-filesystem-dev libboost-test-dev libboost-thread-dev libdb5.3++-dev libminiupnpc-dev libzmq3-dev libqrencode-dev"
 export NO_DEPENDS=1
+export RUN_SECURITY_TESTS=false
```

Comment:
Instead of setting it to false in every config file, why not set it to false by default and only explicitly set it to true in the one config that needs it?

If every config has to specify the value, it sort of defeats the purpose of the default value.

Reply:
Have fixed this up.